src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
```

Comment:
You can just write return without object declaration:
```suggestion
return {
    result: 0,
```

Reply:
Good point, thanks)